ComponentKitTests/CKComponentContextTests.mm

```diff
@@ -69,4 +73,81 @@ - (void)testTriplyNestedComponentContextWithNilMiddleValueCorrectlyRestoresOuter
   XCTAssertTrue(CKComponentContext<NSObject>::get() == outer);
 }
 
+- (void)testFetchingAllComponentContextItemsReturnsArray
```

Comment:
`testFetchingAllComponentContextItemsReturnsObjects`? It isn't an array anymore.